mod/groups/languages/en.php

```diff
@@ -134,6 +134,7 @@
 	'groups:joinrequestmade' => 'Requested to join group',
 	'groups:joined' => 'Successfully joined group!',
 	'groups:left' => 'Successfully left group',
+	'groups:requestexists' => 'You alredy requested join this group, please wait the group owner to respond.',
```

Comment:
`You already requested to join this group. Please wait for the group owner to respond.`